tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -2080,4 +2130,127 @@ namespace tuplex {
         Logger::instance().defaultLogger().info("writing output took " + std::to_string(timer.time()) + "s");
         tstage->setFileResult(ecounts);
     }
+
+    void trimPartitionsToLimit(std::vector<Partition *> &partitions,
```

Comment:
move function maybe to different file? E.g., something like PartitionUtils.h/.cc or so?